src/main/java/uk/gov/pay/webhooks/ledger/model/SettlementSummary.java

```diff
@@ -16,17 +16,17 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class SettlementSummary {
 
-    @JsonDeserialize(using = ApiResponseDateTimeDeserializer.class)
```

Comment:
What's the difference between this new deserialiser and the existing [InstantDeserializer](https://github.com/alphagov/pay-webhooks/blob/e8883fb449daaddb6563b1833438497422e8cdb0/src/main/java/uk/gov/pay/webhooks/util/InstantDeserializer.java)?

Reply:
The main difference is that I did not know `InstantDeserializer` exists! I’ll update the code to use that.